histogram/src/histogram.rs

```diff
@@ -303,11 +301,67 @@ impl Histogram {
         Ok(result)
     }
 
+    /// Creates a copy of this `Histogram` by loading all the values.
+    pub fn load(&self) -> Self {
```

Comment:
Maybe call this `deepcopy` ? By name I'd almost expect `load` to have the same behavior as `store`.

In terms of implementation, can this simply be done with `Histogram::new()` plus `Histogram::store()`? Any. difference at runtime?

Reply:
I should've use the existing clone impl. Removed.